tools/rdm/TestMixins.py

```diff
@@ -317,6 +317,9 @@ class SetWithDataMixin(ResponderTestFixture):
   def Test(self):
     results = [
       self.NackSetResult(RDMNack.NR_FORMAT_ERROR),
+      # TODO(Peter): Fix this, ideally we change behaviour based on past
+      # support of the PID
+      self.NackSetResult(RDMNack.NR_UNSUPPORTED_COMMAND_CLASS),
```

Comment:
Think of e.g device hours, a responder may not support setting it, in which case this response is valid, depending on the order packet format and command class are checked.

The future extension is to remember if we succeeded on a normal SetDeviceHours, pass that in and advisory/warn here that the normal one succeeded and this didn't fail with NR_FORMAT_ERROR.

I can add a warning/advisory to these in the interim saying this should really be an NR_FORMAT_ERROR if you'd like, but I don't think that's correct if SET isn't supported generally for the PID.